api_server/migrations/cryoetdataportal/1722635643562_alter_comments/up.sql

```diff
@@ -0,0 +1,74 @@
+comment on table "public"."datasets" is E'Metadata for a dataset';
+comment on column "public"."datasets"."cell_component_id" is E'If the dataset focuses on a specific part of a cell, the subset is included here';
+comment on column "public"."datasets"."cell_strain_id" is E'Link to more information about the cell strain';
+comment on column "public"."datasets"."related_database_links" is E'If a CryoET dataset is also deposited into another database, e.g. EMPIAR, enter the database identifier here (e.g.https://www.ebi.ac.uk/empiar/EMPIAR-12345/). Use a comma to separate multiple links.';
+
+comment on table "public"."dataset_authors" is E'Metadata for authors of a dataset';
+comment on column "public"."dataset_authors"."dataset_id" is E'Numeric identifier for the dataset this author corresponds to';
+comment on column "public"."dataset_authors"."id" is E'A numeric identifier for this author';
```

Comment:
Could we add a "(May change!)" to this identifier? As this could potentially change, and we don't want the user to have this hardcoded.

Reply:
Done - push incoming. Let me know if there are any other IDs this should apply to that were previously missed.

Reply:
All ids other than the dataset and deposition id could potentially change. Lemme double check if any other ids need that update.

Reply:
For `annotation_authors.id`, I see `this may change!` after getting the schema from the endpoint through introspection.

I did catch a missing `may change!` in `dataset_funding.id`, so added that.

Reply:
Sorry, I was going by the comments on my local graphql. Which strangely has the comment "Numeric identifier for this annotation author (this may not change!)". Must have been a glitch on my end. 
![Screenshot 2024-08-08 at 10 41 23 AM](https://github.com/user-attachments/assets/44470e9a-2600-41d2-81cc-0dd9b7b652f1)